src/components/elements/AccordionItem/AccordionItem.tsx

```diff
@@ -21,25 +21,30 @@ interface AccordionItemProps {
  */
 const AccordionItem = ({ title, def, skillList }: AccordionItemProps) => {
   // State Declarations
-  const [isActive, setIsActive] = useState(def ? true : false)
+  const [isActive, setIsActive] = React.useState(def ? true : false)

```

Comment:
Doublecheck that using 'React.useState' is proper instead of destructuring in components.

Reply:
Going with useState for now until I get told better.